packages/ui/src/components/Avatar.tsx

```diff
@@ -327,9 +328,21 @@ export const SigilAvatar = React.memo(function SigilAvatarComponent({
     if (size && size !== 'custom') {
       return getTokenValue(size);
     } else {
+      if (isWeb && (props.width || props.height)) {
+        // Sigil size must be a number
+        if (
+          typeof props.width === 'string' ||
+          typeof props.height === 'string'
+        ) {
+          return 20;
```

Comment:
do we always want 20 in this case? I don't see any usages of `props.width` or `props.height` being sent as strings

Reply:
I'm not sure about 20, it's just what we were returning by default before (on line 343). Not sure what you mean on the second part of this.

Reply:
Resolved out of band.